src/edu/stanford/nlp/sempre/Main.java

```diff
@@ -22,15 +25,18 @@ public void run() {
     Learner learner = new Learner(builder.parser, builder.params, dataset);
     learner.learn();
 
-    if (server) {
-      Master master = new Master(builder);
-      Server server = new Server(master);
-      server.run();
-    }
-
-    if (interactive) {
-      Master master = new Master(builder);
-      master.runInteractivePrompt();
+    if (server || interactive) {
+      try {
+        Class<?> masterClass = Class.forName(masterType);
```

Comment:
currently, the Master constructor takes a Builder as an argument and not supported by Utils.newInstanceHard() as far as I can tell.